CHANGELOG.md

```diff
@@ -1,3 +1,155 @@
+# 84.4.0
+
+In this minor release we added additional dynamic type support for Buttons, Chips, and List, as well as additional tests and bug fixes.
```

Comment:
Similarly here, can we reword this summary to emphasize that we are expanding support with new features, rather than adding dynamic type support at all?

Reply:
“Support for custom Dynamic Type scaling curves” might help clarify this